Js/index.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable consistent-return */
+const myForm = document.querySelector('form');
+const ply1 = document.getElementById('pl1');
+const ply2 = document.getElementById('pl2');
+const contentField = document.querySelector('.content');
+const input1 = document.getElementById('p1');
+const input2 = document.getElementById('p2');
+const replay = document.getElementById('replay');
+const reload = document.getElementById('reload');
+const gameBoard = document.getElementById('board');
+const winner = document.getElementById('winner');
+
+const moves = ['', '', '', '', '', '', '', '', ''];
+const winningCases = [
+  [0, 1, 2],
+  [0, 4, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [2, 4, 6],
+  [3, 4, 5],
+  [6, 7, 8],
+];
```

Comment:
I think all these can be called global variables right?
In the `Odin` instructions, you have been told to minimize the global variables in this project.
Note: Once you make use of `Factory Functions` and `The Module Pattern` you don't need to have any global variables in your code.

![image](https://user-images.githubusercontent.com/31897434/88721399-4ed49480-d11e-11ea-940f-93cb24c62cfb.png)